packages/fast-components-react-base/src/number-field/number-field.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import {
+    NumberFieldHandledProps,
+    NumberFieldManagedClasses,
+    NumberFieldProps,
+    NumberFieldUnhandledProps,
+} from "./number-field.props";
+import {
+    ManagedClasses,
+    NumberFieldClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { TextFieldType } from "../text-field/index";
+
+class NumberField extends Foundation<
+    NumberFieldHandledProps,
+    NumberFieldUnhandledProps,
+    {}
+> {
+    public static defaultProps: Partial<NumberFieldProps> = {};
+
+    public static displayName: string = "NumberField";
+
+    protected handledProps: HandledProps<NumberFieldHandledProps> = {
+        managedClasses: void 0,
+        disabled: void 0,
+        min: void 0,
+        max: void 0,
+        name: void 0,
+        placeholder: void 0,
+        step: void 0,
+        readOnly: void 0,
+        required: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLDivElement> {
+        return (
+            <input
+                {...this.unhandledProps()}
+                className={this.generateClassNames()}
+                type={TextFieldType.number}
+                {...this.assignAttributes()}
+            />
+        );
+    }
+
+    /**
+     * Generates class names
+     */
+    protected generateClassNames(): string {
+        return super.generateClassNames(get(this.props, "managedClasses.numberField"));
+    }
+
+    /**
+     * Generates attributes based on props
+     */
+    private assignAttributes(): Partial<NumberFieldHandledProps> {
+        const attributes: Partial<NumberFieldHandledProps> = {};
+
+        if (typeof this.props.disabled === "boolean") {
+            attributes.disabled = true;
+        }
+
+        if (typeof this.props.max === "number") {
+            attributes.max = this.props.max;
+        }
+
+        if (typeof this.props.min === "number") {
+            attributes.min = this.props.min;
+        }

```

Comment:
Should this be able to handle negative values? It currently doesn't. I can manually type negative values in the input but cannot in the Configuration panel

Reply:
It should... I will look into it.

Reply:
Everything seems to be working as expected. I can set a negative min in the configurator and it functions as expected. Can you look into your issue again?

Reply:
I still can't set a negative value in the configuration panel... Its probably not a blocker or even an issue with your code. Probably an issue in fast form

Reply:
What browser are you using? I am able to set negative values in the form generator no problem:

<img width="1746" alt="screen shot 2019-02-18 at 3 48 43 pm" src="https://user-images.githubusercontent.com/22480660/52981448-b49ee780-3394-11e9-849b-998838de9df4.png">

Reply:
So weird... I'm using Chrome

Reply:
Anyone else seeing this?

Reply:
Going to resolve for now, but if someone else runs into this issue, I will reopen.